spec/spec_helper.rb

```diff
@@ -1,6 +1,7 @@
 $LOAD_PATH.unshift('./lib/')
 
 ProcessStatusStub = Struct.new(:exitstatus)
-$STATUS_ZERO = ProcessStatusStub.new(0)
-$STATUS_ONE = ProcessStatusStub.new(1)
+STATUS_OK = ProcessStatusStub.new(0)
+STATUS_ERROR = ProcessStatusStub.new(1)
```

Comment:
I agree that they shouldn't be globals, but is OK and ERROR really the right names for them? I'm only asking as they're the exit status codes and 0 means success while 1 is a general (non-specific) error. They were zero and one as they were the corresponding error codes. 

NOTE: I didn't add this code, I have no attachment to the name - I just agree with the current names as they make sense to me.